trabalho-final/MatheusSeabra.md

```diff
@@ -0,0 +1,24 @@
+# Relatório do Trabalho Final - Matheus Seabra
+
+- Nome completo: Matheus Seabra de Carvalho Vieira Coelho
+- Nível: Graduação
+- Matrícula: 201311140021
+- Trabalho em Duplas: ./duplas/Dupla_Hamilton_Matheus.md
+
+## Melhorar documentação
+- Update table links on main READ.ME with respective links #296 (MERGED) (https://github.com/cypress-io/cypress-example-recipes/pull/296)
+- Add 'Community Recipes' table with links to other repositories using cypress (MERGED) #295 (https://github.com/cypress-io/cypress-example-recipes/pull/295)
+
+## Realizar tradução
+- nodejs-pt-br: Add file community.md & Translate it to brazilian portuguese #2303 (IN PROGRESS) (https://github.com/nodejs/nodejs.org/pull/2303)
+
+## Adicionar pequena funcionalidade
+- Feature: add deployment option (OPEN) (https://github.com/the-road-to-graphql/react-graphql-github-apollo/issues/25)
```

Comment:
No caso essa funcionalidade é apenas uma sugestão e não uma implementação, confere? Se não for implementação, não configura como uma tarefa de "Adicionar pequena funcionalidade". Caso isso ocorra, vou pedir que a remova.

Reply:
Eu atualizei de acordo com seus comentários. Não apareceu?

Reply:
Não foi implementação pois estava esperando pelo mantenedor do repositório responder. 

@filipesaraiva

Reply:
Os commits a seguir provam que respondi com os ajustes após os seus comentários:
- https://github.com/gustavopinto/tesl/pull/167/commits/a524100ade88c95878b6770aef07627b9711d139
- https://github.com/gustavopinto/tesl/pull/167/commits/fb185311ae708c5595f38f7d7743b785c81ba39e

Reply:
Beleza, mas você não atualizou o teu PR com esses commits?

Reply:
O corpo do PR foi atualizado, @filipesaraiva

Reply:
@filipesaraiva fico no aguardo da sua resposta